ngraph/frontend/frontend_manager/include/frontend_manager/utils.hpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "frontend_manager_defs.hpp"
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        FRONTEND_API std::string getFrontendLibraryPath();
```

Comment:
it should not be a public API I suppose

Reply:
It should, it is used to get the location of the library. The same is done in inference_engine in file_utils.h

Reply:
Inside IE is NOT a public API.

Public API - what is exposed for users
Developer API - what is exposed for developers.

I suppose you need this function in developer API.